src/dotnet/APIView/APIViewWeb/Models/ReviewCodeFileModel.cs

```diff
@@ -22,7 +22,7 @@ public string Language
 
         public bool HasOriginal { get; set; }
 
-        public DateTime CreationDate { get; set; } = DateTime.Now;
```

Comment:
Without removing this default, the logic to default to the Review date for the old model wouldn't work.

Reply:
Why?

Reply:
How is this value being used?